### ISSUE TITLE
Auto-generate Python functions for the csr read/write functions

### ISSUE THREAD

**Issue by mithro** [im-tomu/micropython#5]

**Comment by mithro** [comment]
@pazzarpj is working on something here -> https://github.com/pazzarpj/micropython-ustubby

**Comment by mattytrentini** [comment]
Yes, it was inspired by Oliver's (@HowManyOliversAreThere) work on the web-based [MicroPython C stub generator](https://mpy-c-gen.oliverrobson.tech/). Both can help generate the boilerplate C required to interface with MicroPython.
